rewrite-core/src/main/java/org/openrewrite/Cursor.java

```diff
@@ -57,7 +57,7 @@ public Cursor getRoot() {
     /**
      * @return true if this cursor is the root of the tree, false otherwise
      */
-    final boolean isRoot() {
+    final public boolean isRoot() {
```

Comment:
Not sure whether this is a desired change

Reply:
You can already check this via a different route: `getCursor().getRoot().equals(getCursor())`. As the `getRoot()` is already public, I don't see why the `isRoot()` should not be public as well.